README.md

```diff
@@ -22,7 +22,7 @@ Modules for running NixOS on the Windows Subsystem for Linux
 3. Import the tarball into WSL:
 
 - ```powershell
-  wsl --import NixOS --version 2 $env:USERPROFILE\NixOS\ nixos-wsl.tar.gz
+  wsl --import NixOS $env:USERPROFILE\NixOS nixos-wsl.tar.gz --version 2
```

Comment:
I think we lost one \ here

Also do the docs need to be changed, too?

Reply:
Yep, sorry for that, it's fixed now. 
Made the adjustments for the `docs/install.md`